### DISCUSSION TITLE
Jira Server Integration

### DISCUSSION THREAD

**Discussion by mattkrick** [ParabolInc/parabol#5445] [New features]
Today, Parabol only supports Jira Cloud.
Some customers use a self-hosted instance of Jira, which is called Jira Server.
This is for those customers.

See https://github.com/ParabolInc/parabol/issues/4915

**Comment by enriquesanchez** [comment]
A Jira Server Integration was the most requested integration from the [Parabol Customer Feedback Survey](https://parabol.slack.com/archives/C08FL5UE7/p1633422567280700) 🔒.

**Reply by deepchags** [reply]
Hi @enriquesanchez @mattkrick when is the Jira Server integration scheduled in the roadmap?

**Reply by enriquesanchez** [reply]
Hi @deepchags!

It is a team priority and we're hoping to have an update soon.

**Comment by drewhousman1010** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/68862701) 🔒  writes:

> OK - so the only way to import issues would be through Jira cloud? We currently have Jira on premise - so I cannot connect to our Jira account - is there another way to import issues?

**Comment by jordanh** [comment]
This is live in production now!
